### DISCUSSION TITLE
Wallet don't receive XCH

### DISCUSSION THREAD

**Discussion by reza646464** [Chia-Network/chia-blockchain#8287] [Support]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop**

- OS: 
- OS Version/Flavor: 
- CPU: 

**Additional context**

**Comment by reza646464** [comment]
hi all
i send some XCH3 time to my wallet but i don't get it after tow week

**Comment by vOBLA86** [comment]
Same here, I haven't received a pool payout, and besides that "chia wallet show" shows me 0.0 XCH

**Comment by keliew** [comment]
> Same here, I haven't received a pool payout, and besides that "chia wallet show" shows me 0.0 XCH

check with the pool?

**Comment by keliew** [comment]
> hi all
> i send some XCH3 time to my wallet but i don't get it after tow week

the sender confirmed that it left his/her wallet?

**Comment by reza646464** [comment]
yes
i am sender

**Reply by loppefaaret** [reply]
try and take the receive address of the transaction, and look it up on either chiaexplorer.com or xchscan.com and verify that it is committed to the blockchain - also verify that the receiving wallet is fully synced

**Comment by BrandtH22** [comment]
@reza646464 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
